test/tools/paths.d

```diff
@@ -0,0 +1,76 @@
+module tools.paths;
+
+import std.file : exists;
+import std.path : buildNormalizedPath, buildPath, dirName, setExtension;
+import std.process : environment;
+
+version (Posix)
+    enum exeExtension = "";
+else version (Windows)
+    enum exeExtension = ".exe";
+
+version(Windows)
+    enum os ="windows";
```

Comment:
Nit: DStyle requires spacing.

Reply:
Hmm, not sure what happened there. I'll fix that.

Reply:
Fixed.